src/runtime/node/trace_events/index.ts

```diff
@@ -0,0 +1,13 @@
+import type trace_events from "node:trace_events";
+import { Tracing } from "./tracing";
+
+export const createTracing: typeof trace_events.createTracing = function () {
+  return new Tracing();
+};
+export const getEnabledCategories: typeof trace_events.getEnabledCategories =
+  () => undefined;
```

Comment:
(not sure but by name i guess desired value is empty array? [])

Reply:
I figured that too, but oddly it's a comma-delimited string:

![image](https://github.com/unjs/unenv/assets/204386/82a91528-ea04-4f9f-86d6-755db98c4516)

Reply:
Thanks for checking. Shall we return empty string then?